rest-api/src/main/java/uk/gov/gchq/gaffer/rest/service/SimpleOperationService.java

```diff
@@ -76,6 +86,38 @@ public Object execute(final OperationChain opChain) {
         return execute(opChain, false);
     }
 
+    @SuppressFBWarnings
+    @Override
+    public ChunkedOutput<String> executeChunked(final OperationChain<CloseableIterable<Element>> opChain) {
+
+        // Create chunked output instance
+        final ChunkedOutput<String> output = new ChunkedOutput<>(String.class);
+
+        // Execute the operation chain and convert to a list
+        final CloseableIterator<Element> objects = execute(opChain, false).iterator();
+
+        final List<Element> elements = asList(Iterators.toArray(objects, Element.class));
```

Comment:
This is going to load all the results into memory. It would be better if we could lazily add them to the chunked output.